includes/classes/blocks/class-template-parts.php

```
@@ -185,45 +177,44 @@ protected function maybe_create_template_part( $slug, $args, $theme ) {
 		// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
 		$content = file_get_contents( $file );
 
-		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-			error_log( sprintf( 'Template Parts: Read %d bytes from %s', strlen( $content ), $slug ) );
-		}
-
 		// Check if template part already exists.
-		$existing = get_posts(
+		// Note: We must use tax_query WITHOUT the 'name' parameter because
+		// WordPress ignores tax_query when 'name' is present in the query args.
+		$query = new \WP_Query(
```

PHPMD (2.15.0)

Missing class import via use statement (line '192', column '16'). (undefined)

(MissingImport)